data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+	npc save disable
+		to spawn
+			has "unfettered: defeated Si-Ra"
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle Prototype (Black)" "Tejolis"
+		personality disables staying heroic target launching
+		conversation
+			action
+				"reputation: Hai (Unfettered)" += 100
+				event "shield beetles accessible"
+			`Now that the fight is over, they both hail you, and He-Yi speaks first. "You sure know how to fight! I feel now maybe we are the ones who need training. I feel like it is only fair you give us tips this time."`
+			choice
+				`	"You were overconfident, and that's what let you down."`
+					goto mistake
+				`	"A good fight, but you just had no chance against my superior firepower."`
+				`	"You lacked coordination with each other."`
+					goto mistake
+				`	"Nothing specific. Well fought to both of you."`
+					goto train
+			`	He-Yi answer first. "It looks like that was indeed the case. Sheer strength may be enough to defeat nearly everything in your way, but is it fun? Is it strategic? I try not to rely on that tactic too often."`
+			`	Sa-Ri then speaks. "Well captain, I sure wish I had the money for such a fleet. I could barely afford this ship, and it will need repairs again now."`
+				goto train
+			label mistake
+			`	He-Yi answers first. "Yes, that is true. We'll try not to repeat that mistake again, Captain."`
+			`	Sa-Ri then speaks. "Next time it will not be so easy, Captain!"`
+			label train
+			`	"The war will start soon, but there is still some time to train. Use that time well, Captain. We have also given you access to our mighty Shield Beetles, should you need it."`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		log `Officially joined the Unfettered Militia and received a license to purchase their heavy ships.`
+		"salary: Unfettered" = 5000
+		set "license: True Unfettered"
+		event "joined the unfettered"
+		clear "unfettered: defeated Si-Ra"
+		#"reputation: Wanderer" <?= -1000
+		conversation
+			`He Yi joins you on your ship after having parked his. "We will notify you once you may be of more help to the war effort. In the meantime, we have increased your salary to 5,000 credits so you can muster an even larger fleet. Also, from now on, you will officially have the permission - nay, the obligation - to proudly wear our colors. We have informed the Hai of your new colors so that they will recognize you and not be hostile to you. We cannot say the same for the Wanderers, though, so be careful."`
+			`	He then draws his long sword, which has the same appearance as the keystone pendants popular among his race, and he hands it to you. "You have also deserved the right to have this. It is a proof of our trust toward you and means you will have access to all of our ships. Whenever a warlord is defeated in battle, he has to surrender his sword to the one who vanquished him. You must be burning with questions about my ship. I am prepared to answer a few."`
```

Comment:
If `<first>` is `burning with questions` about the ship, let's first draw attention to the fact that it's different at all.

Reply:
You did just fight it in space, but  you're right that there could be a mention of it